tests/test_utils.py

```diff
@@ -11,19 +10,18 @@ def test_get_bugs():
     assert(isinstance(bugs, list))
 
 
-def test_mkdir():
-    d = TemporaryDirectory()
-    direc_path = d.name + "/test"
-    utils.mkdir(direc_path)
-    assert(os.path.isdir(direc_path))
-    utils.mkdir(direc_path)
+def test_mkdir(tmpdir):
+    dir_path = tmpdir.strpath + "/test"
+    utils.mkdir(dir_path)
```

Comment:
Changing the name of the variable is unnecessary and leads to changes everywhere, it's better to leave it as it was